src/NewsService.Validation/CreateNewsRequestValidator.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentValidation;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Common;
+using LT.DigitalOffice.NewsService.Models.Dto.Requests;
+using LT.DigitalOffice.NewsService.Validation.Interfaces;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.NewsService.Validation
+{
+  public class CreateNewsRequestValidator : AbstractValidator<CreateNewsRequest>, ICreateNewsRequestValidator
+  {
+    private readonly IRequestClient<ICheckUsersExistence> _rcCheckUsersExistence;
+    private readonly IRequestClient<ICheckDepartmentsExistence> _rcCheckDepartmentsExistence;
+    private readonly ILogger<CreateNewsRequestValidator> _logger;
+
+    public CreateNewsRequestValidator(
+      IRequestClient<ICheckUsersExistence> rcCheckUsersExistence,
+      IRequestClient<ICheckDepartmentsExistence> rcCheckDepartmentsExistence,
+      ILogger<CreateNewsRequestValidator> logger)
+    {
+      _rcCheckUsersExistence = rcCheckUsersExistence;
+      _rcCheckDepartmentsExistence = rcCheckDepartmentsExistence;
+      _logger = logger;
+
+      RuleFor(news => news.AuthorId)
+        .NotEmpty().WithMessage("AuthorId must not be empty.")
+        .MustAsync(async (authorId, cancellation) => await CheckUserExistence(new List<Guid>() { authorId }))
+        .WithMessage("This author doesn't exist.");
+
+      RuleFor(news => news.Pseudonym)
+        .MaximumLength(50).WithMessage("Pseudonym is too long.");
+
+      RuleFor(news => news.Subject)
+        .NotEmpty().WithMessage("Subject must not be empty.")
+        .MaximumLength(120).WithMessage("News subject is too long.");
```

Comment:
we can not now set max length, remove it

Reply:
done